src/NodaTime/CalendarSystem.cs

```diff
@@ -232,10 +236,31 @@ public static CalendarSystem ForId(string id)
         /// <returns>A Hebrew calendar system for the given month numbering.</returns>
         public static CalendarSystem GetHebrewCalendar(HebrewMonthNumbering monthNumbering)
         {
-            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int) monthNumbering, 1, 2);
-            return HebrewCalendars.ByMonthNumbering[((int) monthNumbering) - 1];
+            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int)monthNumbering, 1, 2);
```

Comment:
Just to give a bit more help here - if you use "git reset HEAD~1" to reset to the original code but keep your working copy as it is, you can then create another commit containing just the relevant changes, and use "git push -f" to replace the existing commit in this PR.

If all of this ends up being simply too much hassle, I can sort it out later. I definitely don't want that to block the feature :)